tapdb/sqlc/migrations/000007_universe.up.sql

```diff
@@ -80,8 +80,7 @@ CREATE VIEW universe_stats AS
         COUNT(CASE WHEN u.event_type = 'SYNC' THEN 1 ELSE NULL END) AS total_asset_syncs,
         COUNT(CASE WHEN u.event_type = 'NEW_PROOF' THEN 1 ELSE NULL END) AS total_asset_proofs,
         roots.asset_id,
-        roots.group_key,
-        roots.namespace_root
```

Comment:
It wasn't used elsewhere?

Reply:
I guess it was just in the list for completeness, but the actual change here is that it's removed from the `GROUP BY` clause.